src/harness/testcases/testdata/sas_impl_record_0.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "sas_impl/0/0",
+  "name": "Sas Implementation Record",
+  "administratorId": "sas_admin/sas/0",
```

Comment:
*sigh* Versioning is such a headache. Let's try to keep to the latest protocol specs since the changes are minimal, they'll be finalized "soon", and cert will be based on those anyway. Once they are actually finalized we might want to do a quick sweep of the code to check for parts that need to be updated (although changes in the test spec will probably tell us where to look).